cms/djangoapps/contentstore/tasks.py

```diff
@@ -10,7 +10,6 @@
 from datetime import datetime  # lint-amnesty, pylint: disable=wrong-import-order
 from tempfile import NamedTemporaryFile, mkdtemp  # lint-amnesty, pylint: disable=wrong-import-order
 
-import olxcleaner
```

Comment:
Why are the import order changes needed in a PR that didn't upgrade pylint or isort?

Reply:
Not really sure myself. These failures started coming on Github actions quality, but on the other hand, Jenkins doesn't mind these changes and gives any error there.

Reply:
It looks like these are all related to codejail and olxcleaner, which are editable installs from GitHub.  Are these changes still needed after you added the `--src` option to `pip install`?  At any rate, it probably has something to do with where pylint/isort are finding those packages on the filesystem.  Running isort locally reports problems after these changes but not before.

Reply:
Yes, they're still needed for some reason, see the result if I try to revert these changes https://github.com/edx/edx-platform/runs/2929499074?check_suite_focus=true#step:10:94,
and I was not able to reproduce this issue locally. On my local paver quality checks pass even with these changes. So I am still investigating the reason for this.